torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -388,16 +388,16 @@ void scheduleReduction(
       //                --------------------------------
       //                Reduction Dimensions
       red_tv->split(1, rparams.loop_unroll);
-      red_tv->split(1, rparams.lparams.bdimx());
+      red_tv->split(1, NamedScalar::getParallelDim(ParallelType::TIDx));
```

Comment:
Don't have a strong opinion on this:
At this stage, we already have a static number for split in `rparams.lparams`.

Are we using symbolic split here to support tuning of launch parameters?

Reply:
@kevinstephano this is the only real change that I notice, which could be related to the reduction shmoo test failure.
I don't see how it is wrong but just in case you got a better idea.

Reply:
Scratch this ^^^

Was pointed out in slack that symbolic split is necessary in order to reuse kernel with various sizes.